video/out/vulkan/context.c

```diff
@@ -266,8 +266,9 @@ bool ra_vk_ctx_init(struct ra_ctx *ctx, struct mpvk_ctx *vk,
         .present_mode = preferred_mode,
         .swapchain_depth = ctx->vo->opts->swapchain_depth,
         // mpv already handles resize events, so gracefully allow suboptimal
-        // swapchains to exist in order to make resizing even smoother
-        .allow_suboptimal = true,
+        // swapchains to exist in order to make resizing even smoother on some
+        // platforms.
+        .allow_suboptimal = p->params.allow_suboptimal,
```

Comment:
Maybe it should just be a configuration option?

Reply:
I thought it made more sense to do it this way but I can change it if that's what's preferred